src/doc/tutorials/PythonExpressions.rst

```diff
@@ -0,0 +1,202 @@
+5. Click *New* in the *Expression list* to create a new expression.
+6. Change the *Name* in the *Definition* section to "MyExpression".
+7. Click on the *Python expression editor* tab in the *Definition* section.
+8. Select *Insert variable...->Scalars->d* to add *d* to the *Arguments* text field.
+9. Click *Load script->Template->Simple filter* to add a template of a Python filter in the *Python expression script* editor. 
+
+.. figure:: images/PythonExpression-GUI1.png
+
+   The Expressions window with the simple filter template
+
+At this point you modify the template to create your expression.
+There are a couple of ways in which you can develop your script.
+In either case you would make modifications to the script and then try it out using the Pseudocolor plot.
+Changes to the script can be made either by modifying the script in the *Python expression script* editor or modifying it in an external text editor and then reloading the script.
+Generaly speaking, modifying the script in the *Python expression script* editor is easier than doing it in an external text editor except that it is difficult to tell how many spaces are at the beginning of the line since the editor doesn't use a fixed width font.
+
+Developing the script in the *Python expression script* editor
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following steps can be used to iteratively develop your script using the *Python expression script* editor.
+
+1. Edit the script.
+2. Click the *Apply* button.
+3. Go to *Variables->d* to change the variable to *d*.
+   The first time you try your script this won't be necessary since the variable is already *d*.
+4. Go to *Variables->MyExpression* to change the variable to "MyExpression" and execute the script.
+
+Developing the script in a text editor
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
+
+The following steps can be used to iteratively develop your script using a text editor.
+
+Before you can modify the script you will need to save it.
+
+1. Click *Save script* to save the script.
+
+Now you are ready to modify the script.
+
+2. Edit the script with your favorit editor.
+3. Go to *Load script->File* to load the script.
+4. Click the *Apply* button.
+5. Go to *Variables->d* to change the variable to *d*.
+   The first time you try your script this won't be necessary since the variable is already *d*.
+6. Go to *Variables->MyExpression* to change the variable to "MyExpression" and execute the script.
+ 
+Python Expression Example 1
+---------------------------
+
+This example will operate on 2D meshes and takes the distance around the edges of each cell and multiplies it by the value of the cell.
+It demonstrates accessing the coordinates and topology of the mesh as well as a variable.
+
+Here is the example script.
+
+.. literalinclude:: ../../test/tests/hybrid/python_example_1.vpe
+   :lines: 11-48
+
+Let's start off by creating a Pseudocolor plot from the expression.
```

Comment:
FWIW, our documentation also says to [avoid contractions](https://www.e-education.psu.edu/styleforstudents/c1_p15.html). Now, its certainly possible that not everyone read and/or signed off on all the bullets in our [quick ref](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/ContributingToDocumentation/index.html#quick-reference) section but there are notes there about trying not to use contractions. My preference is to avoid them.